### PR TITLE
Small backported changes to Food.pm

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: lib/ProductOpener/Food.pm [hunk 1]
```diff
@@ -3785,6 +3785,18 @@ sub mmoll_to_unit {
...
+		de => "Internationale Bittereinheit",
+		en => "International Bittering Unit",
+		pt => "Medida do amargor internacional",
+		ru => "Международная единица горечи",
+		unit => "",
+	},
+	"alcohol-units" => {
+		de => "Alkoholeinheit",
+		fr => "Unité d'alcool",
+	unit => "'",
```
Comment (stephanegigandet) [inline lib/ProductOpener/Food.pm:3798]: ' ?

Comment (teolemon) [inline lib/ProductOpener/Food.pm:3798]:
```suggestion
	unit => "",
```

Comment (teolemon) [reply lib/ProductOpener/Food.pm:3798]: thanks

File: lib/ProductOpener/Food.pm [hunk 2]
```diff
@@ -3785,6 +3785,18 @@ sub mmoll_to_unit {
...
 	},
+	"ibu" => {
+		de => "Internationale Bittereinheit",
+		en => "International Bittering Unit",
+		pt => "Medida do amargor internacional",
+		ru => "Международная единица горечи",
+		unit => "",
+	},
+	"alcohol-units" => {
+		de => "Alkoholeinheit",
```
Comment (stephanegigandet) [inline lib/ProductOpener/Food.pm:3796]:
```suggestion
		en => "Alcohol units",
		de => "Alkoholeinheit",
```

File: lib/ProductOpener/Food.pm [hunk 3]
```diff
@@ -3785,6 +3785,18 @@ sub mmoll_to_unit {
...
+		fr => "Unité d'alcool",
```
Comment (stephanegigandet) [inline lib/ProductOpener/Food.pm:3797]: Should probably be plural. (same for German if it's not plural)

File: lib/ProductOpener/Food.pm (hunk 1 above)
Comment (teolemon) [reply lib/ProductOpener/Food.pm:3798]: I'm not even sure who added them, the id was not even compliant in the original PR

File: lib/ProductOpener/Food.pm [hunk 4]
```diff
@@ -3785,6 +3785,19 @@ sub mmoll_to_unit {
...
+		de => "Internationale Bittereinheit",
+		en => "International Bittering Unit",
+		pt => "Medida do amargor internacional",
+		ru => "Международная единица горечи",
+		unit => "",
+	},
+	"alcohol-units" => {
+		en => "Alcohol units",
+		de => "Alkoholeinheit",
+		fr => "Unité d'alcool",
```
Comment (teolemon) [inline lib/ProductOpener/Food.pm:3798]:
```suggestion
		en => "Alcohol units",
		de => "Alkoholeinheiten",
		fr => "Unités d'alcool",
```

File: lib/ProductOpener/Food.pm (hunk 3 above)
Comment (teolemon) [reply lib/ProductOpener/Food.pm]: fixed

**Comment by stephanegigandet** [comment]
We'll need to be careful that the serving size is indicated if there are alcohol units. Let's see how it goes.

**Pull request merged**